django_airbrake/middleware.py

```diff
@@ -8,4 +8,8 @@ def __init__(self):
 
     def process_exception(self, request, exception):
         if hasattr(settings, 'AIRBRAKE') and not settings.AIRBRAKE.get('DISABLE', False):
-            self.client.notify(exception=exception, request=request)
+            try:
+                self.client.notify(exception=exception, request=request)
+            # Ignore failures from issues like rate limiting
+            except:
+                pass
```

Comment:
I'm not sure why I even got an email about this PR but I'll comment... I don't know this code at all but my guess is that `exception` here is the actual exception that was raised. Instead of `pass`, could we just `raise exception` so we get the actual exception?